packages/cli/src/utils/banner.js

```diff
@@ -1,10 +1,10 @@
 import consola from 'consola'
-import prettyBytes from 'pretty-bytes'
 import env from 'std-env'
 import chalk from 'chalk'
 import { successBox } from './formatting'
+import { getMemoryUsage } from './memory'
 
-export function showBanner(nuxt) {
+export function showBanner(nuxt, showMemoryUsage = true) {
```

Comment:
Banner will be always shown by CLI before any build happening, so it may be more consistent to show it only as a info message not in banner.

Reply:
showBanner seems only to be used in `dev` and `start`, for start I think it would look strange to print an additional line

Reply:
OK. Makes sense